Makefile

```
@@ -6,11 +6,37 @@ ifeq ($(OS),Windows_NT)
 	extension = .exe
 endif
 
-.DEFAULT_GOAL := build
+# Default target
+all: fmt lint test benchmark
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)